biodatasets/pubmed_qa/pubmed_qa.py

```diff
@@ -0,0 +1,245 @@
+            schema="bigbio_qa",
+            subset_id="pqau",
+        ),
+    ] + [
+        # PQAL Source Schema        
+        BigBioConfig(
+            name=f"pqal_fold{i}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description="PubmedQA labeled source schema",
+            schema="source",
+            subset_id=f"pqal_fold{i}"
+        ) for i in range(10)
+    ] + [
+        # PQAL BigBio Schema
+        BigBioConfig(
+            name=f"pqal_fold{i}_bigbio_qa",
+            version=datasets.Version(_BIGBIO_VERSION),
+            description="PubmedQA labeled BigBio schema",
+            schema="bigbio_qa",
+            subset_id=f"pqal_fold{i}"
+        ) for i in range(10)
+    ]
+
+    DEFAULT_CONFIG_NAME = "pqaa_source"
+
+    def _info(self):
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "QUESTION": datasets.Value("string"),
+                    "CONTEXTS": datasets.Sequence(datasets.Value("string")),
+                    "LABELS": datasets.Sequence(datasets.Value("string")),
+                    "MESHES": datasets.Sequence(datasets.Value("string")),
+                    "YEAR": datasets.Value("string"),
+                    "reasoning_required_pred": datasets.Value("string"),
+                    "reasoning_free_pred": datasets.Value("string"),
+                    "final_decision": datasets.Value("string"),
+                    "LONG_ANSWER": datasets.Value("string")
+                },
+            )
+        elif self.config.schema == "bigbio_qa":
+            features = schemas.qa_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        subset_id = self.config.subset_id
+        if 'pqal' in subset_id:
+            subset_id = 'pqal'
+
+        urls = _URLS[subset_id]
+        data_dir = Path(dl_manager.download_and_extract(urls))
+
+        if 'pqal' in self.config.subset_id:
+            return [
+                datasets.SplitGenerator(
+                    name=datasets.Split.TRAIN,
+                    gen_kwargs={
+                        "filepath": data_dir / self.config.subset_id / "train_set.json"
+                    }
+                ),
+                datasets.SplitGenerator(
+                    name=datasets.Split.VALIDATION,
+                    gen_kwargs={
+                        "filepath": data_dir / self.config.subset_id / "dev_set.json"
+                    }
+                ),
+                datasets.SplitGenerator(
+                    name=datasets.Split.TEST,
+                    gen_kwargs={
+                        "filepath": data_dir / "pqal_test_set.json"
+                    }
+                )
+            ]            
```

Comment:
if I'm reading this correctly, I think we don't need to include the cv splits

Reply:
Hi @galtay, thanks for reviewing. The CV split is only defined for train and dev, I was thinking to merge train and validation data or using a single split before, but to strictly follow the split defined in https://github.com/pubmedqa/pubmedqa, we need to split the train and dev sets into 10-folds.

Reply:
ok, I think i see now, so there are train/dev/test splits WITHIN each fold?